arcgis-ios-sdk-samples/Content Display Logic/Controllers/ContentCollectionViewController.swift

```diff
@@ -175,12 +156,21 @@ class ContentCollectionViewController: UIViewController, UICollectionViewDataSou
     
     //item width based on the width of the collection view
     func itemSizeForCollectionViewSize(_ size:CGSize) -> CGSize {
+        let spacing: CGFloat = 10
         //first try for 3 items in a row
-        var width = (size.width - 4*10)/3
-        if width < 150 {    //if too small then go for 2 in a row
-            width = (size.width - 3*10)/2
+        var width = (size.width - 4*spacing)/3
+        if width < 150 {
+            //if too small then go for 2 in a row
+            width = (size.width - 3*spacing)/2
         }
         return CGSize(width: width, height: width)
     }
     
 }
+
+class CustomFlowLayout:UICollectionViewFlowLayout {
+    
+    override func shouldInvalidateLayout(forBoundsChange newBounds: CGRect) -> Bool {
+        return true
```

Comment:
This isn't needed anymore. Could you please remove it, along with our implementation of `ContentCollectionViewController.viewWillTransition(to:with:)`? Unless you can find a case where it is still needed.

Reply:
iOS 10 seems to still need these in order to resize the items when rotating the device. This is only a minor visual nicety and such an edge case that I'm going to take these methods out anyway.